<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageHex">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="textHex"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageGraph"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionGithub"/>
   </widget>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>Operation</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionTest"/>
    <addaction name="actionSerialPort"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Mode">
    <property name="title">
     <string>&amp;Mode</string>
    </property>
    <addaction name="actionHex"/>
    <addaction name="actionGraph"/>
   </widget>
   <addaction name="menuOperation"/>
   <addaction name="menu_Mode"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/open.svg</normaloff>:/resources/icons/open.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/exit.svg</normaloff>:/resources/icons/exit.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/about.svg</normaloff>:/resources/icons/about.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/qt.png</normaloff>:/resources/icons/qt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionGithub">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/github.svg</normaloff>:/resources/icons/github.svg</iconset>
   </property>
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="actionSerialPort">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/serialport.svg</normaloff>:/resources/icons/serialport.svg</iconset>
   </property>
   <property name="text">
    <string>SerialPort</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionHex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/hex.svg</normaloff>:/resources/icons/hex.svg</iconset>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </action>
  <action name="actionGraph">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
